external/implementations/samlconf-ddf-impl/README.md

```diff
@@ -8,4 +8,4 @@
     * Do the same copy-paste steps but copying the **second** `EntityDescriptor` element.
 * If not on localhost, copy DDF's IDP metadata from `https://<hostname>:<port>/services/idp/login/metadata` 
 to the file `<samlconf>/implementations/ddf/ddf-idp-metadata.xml` where `<samlconf>` is the root directory of the test kit distribution.
-* Run `<samlconf>/bin/samlconf` where `<samlconf>` is the root directory of the test kit distribution. (no parameters needed since it points to DDF's implementation by default)
\ No newline at end of file
+* From the root of the saml-conformance GitHub repo, run `deployment/distribution/build/install/samlconf/bin/samlconf -l`. Note: This assumes the project has been succesfully built.
```

Comment:
✏️ "saml-conformance GitHub repo" sounds odd to me for some reason. I think it is because when I hear "GitHub repo" I think of the remote repository, not a local one.

Maybe "From the root of the saml conformance source directory" will sound better to me instead.